src/core/maps/const_occupancy_estimator.h

```diff
@@ -1,12 +1,23 @@
+
 #ifndef __CONST_OCCUPANCY_ESTIMATOR_H
 #define __CONST_OCCUPANCY_ESTIMATOR_H
 
 #include "cell_occupancy_estimator.h"
 
+/**
+ * \brief This class is responsible for strategy of base occupancy estimation.
```

Comment:
Should be smth like this "Implements occupancy estimator strategy that returns probability of being occupaid for a cell depending only on the cell status predicted by a sensor. "